hw/chisel/src/main/scala/snax/xdma/xdmaExtension/DMAExtension.scala

```diff
@@ -98,3 +97,73 @@ abstract class DMAExtension(implicit extensionParam: DMAExtensionParam)
   // When bypass is 1, io.in(1) is connected to bypass signal
   outputMux.io.in(1) <> bypass_data
 }
+
+/** The parent Class for the integration of DMA Extension written in
+  * SystemVerilog Before the integration of the custom extension, please make
+  * sure the following: 1) The SystemVerilog module's top module's IO is strict,
+  * shown as the following example:
+  *
+  * module VerilogMemset #( userCsrNum = 1, dataWidth = 512 ) ( input logic clk,
+  * input logic rst_n, output logic ext_data_i_ready, input logic
+  * ext_data_i_valid, input logic [dataWidth-1:0] ext_data_i_bits, input logic
+  * ext_data_o_ready, output logic ext_data_o_valid, output logic
+  * [dataWidth-1:0] ext_data_o_bits, input logic [31:0]ext_csr_i_0, input logic
+  * ext_start_i, output logic ext_busy_o ); Several reminders for developers:
+  * \1. Two parameters is provided by XDMA generator: userCsrNum and dataWidth,
+  * which is user-definable in HasDMAExtension class. 2. rst_n is the active low
+  * asynchronous reset signal. 3. The input and output has the handshake. When
+  * both are high, the data is regarded as successfully transferred. 4. csr_i is
+  * the CSR signal, and it is an unrolled two-dimensional signal. If userCsrNum
+  * is 2, then there is two csrs: ext_csr_i_0 and ext_csr_i_1.
```

Comment:
Actually, this one too so that it's readable. Put each number in a new line.

```suggestion
  * 1. Two parameters are provided by the XDMA generator: userCsrNum and data width,
  * which is user-definable in the HasDMAExtension class. 
  * 2. rst_n is the active low asynchronous reset signal. 
  * 3. The input and output have the handshake. When both are high, the data is regarded as successfully transferred.
  * 4. csr_i is the CSR signal, and it is an unrolled two-dimensional signal. If userCsrNum
  * is 2, then there are two csrs: ext_csr_i_0 and ext_csr_i_1.
```